### PR TITLE
Add pangenome test data set keys

### PR DESCRIPTION
**Pull request by subwaystation**
## PR checklist

- [X] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

This PR accompanies https://github.com/nf-core/test-datasets/pull/406. It's aim is to promote the test data added there.
Once this PR got approved, I will merge both at the same time.

### CODE REVIEW

**Review by FriederikeHanssen — APPROVED** [review]
lgtm

**Comment by heuermh** [comment]
Looks good to me, but the git history is hard to follow.  Might you be able to rebase to squash commits and force push?

**Comment by subwaystation** [comment]
> Looks good to me, but the git history is hard to follow. Might you be able to rebase to squash commits and force push?

Seem like I can only cherry pick real commits and not these Merges from branch `master` @heuermh 

```
pick 37c5cb49 feat: add original input as optional output channel (#1147)
pick a68c563e Added UMI sub-workflow (#1098)
pick 1f3f2b18 Add new module macrel/contigs (#1109)
pick be852899 Update metabat2 output channels and compression (#1111)
pick 1cc2d0d7 Update seqwish to version 0.7.2
squash dfb644b9 seqwish can work with a comma-separated list of PAFs
pick 0bf40a26 Update seqwish to version 0.7.2 (#1144)
pick edbbbbf4 fix: exclude conda/singularity images from pytest workflow output (#1162)
pick 47a9cf8e Fix transcriptome staging issues on DNAnexus for rsem/prepareference (#1163)
pick 0fafaeeb Revert PR #1147 (#1166)
pick 826a5603 Stage fastq for concat in subfolders to avoid name collision issue (#1107)
pick 4e5406c2 Add helper script to find duplicate test YAML md5s (#1167)
squash 9d9ea83b level with nf-core/modules master branch
```

So I would just hit the `Squash and merge` button anyhow?

**Comment by heuermh** [comment]
> So I would just hit the Squash and merge button anyhow?

Yeah, that might work 😉   Most of the projects I work with use rebasing exclusively so I get somewhat confused by merge commits.

**Review by heuermh — APPROVED** [review]

**Comment by heuermh** [comment]
`"${test_data_dir}/pangenomics/homo_sapiens/pangenome.og"` won't be found, the path would be
`pangenomics/odgi/pangenome.og`, see https://github.com/nf-core/test-datasets/tree/modules/data/pangenomics/odgi

Perhaps the two `odgi` files need to be moved in the nf-core/test-datasets repository?

**Comment by subwaystation** [comment]
Alright @heuermh . Please take a look at https://github.com/nf-core/test-datasets/pull/410 and this PR. The file structure should match now. Thanks for finding this!

**Comment by subwaystation** [comment]
@heuermh Could you please finalize your review? Thanks!

**Review by heuermh — APPROVED** [review]

**Comment by heuermh** [comment]
LGTM, and the test data pull request https://github.com/nf-core/test-datasets/pull/410 has been merged.
